src/components/ProfileTableCell.tsx

```diff
@@ -1,25 +1,28 @@
 import React from 'react'
-import { StyleSheet } from 'react-native'
+import { StyleSheet, TouchableWithoutFeedback } from 'react-native'
 import { PV } from '../resources'
 import { Text, View } from './'
 
 type Props = {
-  title?: string
+  name?: string
+  onPress?: any
 }
 
 export const ProfileTableCell = (props: Props) => {
-  const { title } = props
+  const { name, onPress } = props
 
   return (
-    <View style={styles.wrapper}>
-      <Text style={styles.title}>{title}</Text>
-    </View>
+    <TouchableWithoutFeedback onPress={onPress}>
```

Comment:
If the View inside the Touchable is so that you can render the text and nothing more, wanted to let you know, the <Text> component has an onPress prop on it directly

Reply:
Hmm I'm not sure what you mean by this. I just tried removing Touchable and moved onPress into the View, but then on press the event was never called...am I misunderstanding something?